internal/blockchain/ethereum/ethereum.go

```diff
@@ -1193,8 +1119,45 @@ func (e *Ethereum) getNetworkVersion(ctx context.Context, address string) (int,
 	return strconv.Atoi(output.Output.(string))
 }
 
-func (e *Ethereum) NetworkVersion() int {
-	e.fireflyContract.mux.Lock()
-	defer e.fireflyContract.mux.Unlock()
-	return e.fireflyContract.networkVersion
+func (e *Ethereum) GetAndConvertDeprecatedContractConfig(ctx context.Context) (location *fftypes.JSONAny, fromBlock string, err error) {
+	// Old config (attributes under "ethconnect")
+	address := e.ethconnectConf.GetString(EthconnectConfigInstanceDeprecated)
+	if address != "" {
+		log.L(ctx).Warnf("The %s.%s config key has been deprecated. Please use %s.%s instead",
+			EthconnectConfigKey, EthconnectConfigInstanceDeprecated,
+			FireFlyContractConfigKey, FireFlyContractAddress)
```

Comment:
I'm not sure this accurately describes the "new" config. Should ultimately be something like "Please use namespaces.predefined[].multiparty.contract.location.address" (wow it's long... but still correct I think)